source/includes/kubernetes_extension/_k8_serviceaccounts.md

```diff
@@ -54,6 +54,59 @@ Retrieve a list of all service accounts in a given [environment](#administration
 
 Note that the list is not complete, since it is refering to the [kubernetes api details](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md).
 
+<!-------------------- GET A SERVICE ACCOUNT -------------------->
+
+##### Get a service account
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/a_service/an_environment/serviceaccounts/serviceaccount-name/serviceaccount-namespace?cluster_id=projects/cmc-k8s-enabled-llb/locations/us-central1-a/clusters/standard-cluster-1"
```

Comment:
we should change this to `?cluster_id=:cluster_id"` so it matches the list request